### DISCUSSION TITLE
Google System Method Patched?

### DISCUSSION THREAD

**Discussion by TheAnimemer** [3kh0/ext-remover#743] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I've been using the Google System Method for a little while now. My chromebook restarted recently and now when I click the Github logo following the steps, it opens github in a seperate tab, ruining the exploit. As far as I know that means it no longer works and I wanted to let others know.

**Comment by AshtonDavies** [comment]
What version is it patched?

**Reply by TheAnimemer** [reply]
I am on 118.0.5993.86 (Official Build) (64-bit) 
the platform is 15604.45.0 (Official Build) stable-channel snappy

It could very well be something else maybe a user error and not a patch per se. Whatever the reason, I definetly can't get it to work anymore. It was working Friday, havent tried it again until today when it wouldnt work. Hope this helps!
